gatorgrade/output/output_functions.py

```diff
@@ -1,7 +1,49 @@
-"""Add docstring."""
+"""
+This module is used for storing the main functions requested.
 
+The requested functions are located at the Github Issue Tracker
+for the output team. For instance, functions dealing with percentage
+output, description output, and colorization of text.
+"""
 
-# This class is used for storing the main functions requested
-# from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output,
-# description output, and colorization of text.
+import gator
+
+
+def run_commands_and_return_results(commands_input):
+    """
+    Receive commands and send results to other output methods.
+
+    Commands are received as dictionary of two keys, shell commands / gator
+    commands.
+
+    An Example of input for this function is as follows :
+
+    {'shell': [{'description': 'Run program', 'command': 'mdl'}],
+    'gatorgrader': [['--description', 'do command', 'commandType',
+    '--arg', '1', '--directory', './home', '--file', 'file.py']]}
+    """
+    # pylint: disable=W0703
+    # Get first element in list, which is gatorgrader commands
+    gatorcommands = commands_input.get("gatorgrader")

```

Comment:
I'm going to second what Noor said here, you can also just assume it's there in a try except keyerror block which is faster than testing if the thing exists.

Reply:
Same thing as what I told Noor, the gatorgrader key will always exist.